### PR TITLE
Feat/add twinbuilder example

### PR DESCRIPTION
**Pull request by sushobhit-gupt**
Add a new RC Circuit example for Twin Builder
Add a new Voltage Source Component for Twin Builder

### CODE REVIEW

File: pyaedt/modeler/PrimitivesSimplorer.py [hunk 1]
```diff
@@ -178,6 +178,61 @@ def create_capacitor(self, compname=None, value=50, location=[], angle=0, use_in
...
+        self, compname=None, type="E", amplitude=326, freq=50, location=[], angle=0, use_instance_id_netlist=False
+    ):
+        """Create a voltage source.
+
+        Parameters
+        ----------
+        compname : str, optional
+            Name of the voltage source. The default is ``None``.
+        type  : str, optional
+            Type of the source. The default is ``E``
```
Comment (MaxJPRey) [inline pyaedt/modeler/PrimitivesSimplorer.py:195]:
```suggestion
            Type of the source. The default is ``E``.
```

File: pyaedt/modeler/PrimitivesSimplorer.py [hunk 2]
```diff
@@ -178,6 +178,61 @@ def create_capacitor(self, compname=None, value=50, location=[], angle=0, use_in
...
+        amplitude : float, optional
+            Amplitude of the waveform if periodic. The default is ``326V``
+        freq : float, optional
+            frequency of the periodic waveform. The default is ``50Hz``.
+        location : list of float, optional
+            Position on the X axis and Y axis.
+        angle : float, optional
+            Angle rotation in degrees. The default is ``0``.
```
Comment (MaxJPRey) [inline pyaedt/modeler/PrimitivesSimplorer.py:203]:
```suggestion
            Angle of rotation in degrees. The default is ``0``.
```

File: pyaedt/modeler/PrimitivesSimplorer.py [hunk 3]
```diff
@@ -178,6 +178,61 @@ def create_capacitor(self, compname=None, value=50, location=[], angle=0, use_in
...
+        Parameters
+        ----------
+        compname : str, optional
+            Name of the voltage source. The default is ``None``.
+        type  : str, optional
+            Type of the source. The default is ``E``
+        amplitude : float, optional
+            Amplitude of the waveform if periodic. The default is ``326V``
+        freq : float, optional
+            frequency of the periodic waveform. The default is ``50Hz``.
```
Comment (MaxJPRey) [inline pyaedt/modeler/PrimitivesSimplorer.py:199]:
```suggestion
            Frequency of the periodic waveform. The default is ``50Hz``.
```

File: pyaedt/modeler/PrimitivesSimplorer.py [hunk 4]
```diff
@@ -178,6 +178,61 @@ def create_capacitor(self, compname=None, value=50, location=[], angle=0, use_in
...
+        location : list of float, optional
+            Position on the X axis and Y axis.
+        angle : float, optional
+            Angle rotation in degrees. The default is ``0``.
+        use_instance_id_netlist : bool, optional
+            Whether to use the instance ID in the net list. The default is ``False``.
```
Comment (MaxJPRey) [inline pyaedt/modeler/PrimitivesSimplorer.py:205]:
```suggestion
            Whether to use the instance ID in the net list or not. The default is ``False``.
```

File: examples/07-TwinBuilder/01-RC_Circuit_Example.py [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+from pyaedt import TwinBuilder
+from pyaedt import Desktop
+import os
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Launch Twin Builder
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# This examples launches Twin Builder 2021.2 in graphical mode.
```
Comment (MaxJPRey) [inline examples/07-TwinBuilder/01-RC_Circuit_Example.py:16]:
```suggestion
# This example launches Twin Builder 2021.2 in graphical mode.
```

File: examples/07-TwinBuilder/01-RC_Circuit_Example.py [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+# This examples uses SI units.
```
Comment (MaxJPRey) [inline examples/07-TwinBuilder/01-RC_Circuit_Example.py:18]:
```suggestion
# This example uses SI units.
```

**Review by MaxJPRey — APPROVED** [review]

**Comment by MaxJPRey** [comment]
@sushobhit-gupt 
In each folder containing an example we have a Readme.txt file presenting the purpose of those examples. Maybe you can add it.
Otherwise, it looks good to me and very nice example.

**Comment by maxcapodi78** [comment]
@sushobhit-gupt @MaxJPRey  I agree, we need the readme.txt in the folder. This will be used in the header of the examples in online documentation. You can take as reference one of the other readme files.

**Review by maxcapodi78 — CHANGES_REQUESTED** [review]
Please add the Readme.txt. Then it looks great to me.

**Comment by sushobhit-gupt** [comment]
> Please add the Readme.txt. Then it looks great to me.

@maxcapodi78 @MaxJPRey I have added the new readme from Twin Builder and merged the changes from @MaxJPRey .
Please take a look when you get a chance.

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**